microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -22,89 +22,102 @@ ms.date: 05/04/2021
    limitations under the License.  -->
 # View page resources
 
-Resources are the files that a page needs in order to display correctly.  Examples of resources include:
+Resources are the files that a page needs to display correctly.  Examples of resources include:
 *  CSS files.
 *  JavaScript files.
 *  HTML files.
 *  Image files.
 
 You can view the resources that make up a web page from within several tools, or panels.
 
-This guide assumes that you are familiar with the basics of [web development](https://developer.mozilla.org/docs/Learn) and [Microsoft Edge DevTools](../../devtools-guide-chromium/overview.md).
-
+This guide assumes that you are familiar with the basics of web development described in [Learn web development](https://developer.mozilla.org/docs/Learn) and [Microsoft Edge DevTools](../../devtools-guide-chromium/overview.md).
 
+To open DevTools, right-click the webpage, and then select **Inspect**. Or press `Ctrl`+ `Shift`+ `I` (Windows, Linux) or `Command`+ `Option`+ `I` (macOS).
```

Comment:
This paragraph is confusing when placed here out of any task scenario context.  Maybe move down into some h2 sections.

Reply:
I added this at first, thinking that we wouldn't have to tell a user how to open DevTools in every section. Changed my mind but forgot to delete it.